amo2kinto/amo.py

```diff
@@ -19,6 +19,8 @@ def prepare_amo_records(records, fields):
             else:
                 if isinstance(record[field], six.string_types):
                     record[field] = record[field].strip()
+                elif isinstance(record[field], (int, float)):
+                    record[field] = six.text_type(record[field])
```

Comment:
I think you should add a comment here.
It works only because `severity` and `vulnerability` are in a sub-property called `versionRange`